src/backend/index.ts

```diff
@@ -2,43 +2,41 @@ import { createTransactionsInExternalVendors } from '@/backend/export/exportTran
 import { scrapeFinancialAccountsAndFetchTransactions } from '@/backend/import/importTransactions';
 import moment from 'moment';
 import * as configManager from './configManager/configManager';
-import {
-  BudgetTrackingEvent, BudgetTrackingEventEmitter, EventNames, EventPublisher
-} from './eventEmitters/EventEmitter';
 import { buildCompositeEventPublisher } from './eventEmitters/compositeEventPublisher';
 import { buildConsoleEmitter } from './eventEmitters/consoleEmitter';
 import outputVendors from './export/outputVendors';
 import * as bankScraper from './import/bankScraper';
+import * as Events from './eventEmitters/EventEmitter';
 
 export { CompanyTypes } from 'israeli-bank-scrapers-core';
 export { outputVendors };
 export { configManager };
-export { BudgetTrackingEvent, BudgetTrackingEventEmitter };
+export { Events };
```

Comment:
Trying to export "groups" instead of exporting a lot of functions, types, and references.